### PR TITLE
Completed html-forms exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
https://user-images.githubusercontent.com/122064415/214373124-3c332431-b3fe-4d97-b171-c4c00f8b50e8.mov

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html">
+      <div>Email</div>
```
Comment (JenBowman1) [inline html-forms/index.html]: Please wrap your titles within the form in a label tag !
Ex:  < label for="email-address" > Email </ label >

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <input type="email" name="email" />
+
+      <div>Why is coding cool?</div>
```
Comment (JenBowman1) [inline html-forms/index.html]: see above

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+      <div>Favorite Language (choose one)</div>
+      <input type="radio" name="language" value="html" />HTML
+      <input type="radio" name="language" value="css" />CSS
+      <input type="radio" name="language" value="javascript" />JavaScript
+
+      <div>Would you like to sign up for more information</div>
+      <input type="checkbox" name="subsrcibe" value="sign-me-up" />Sign me up!
+
+      <div>Comments</div>
```
Comment (JenBowman1) [inline html-forms/index.html]: see above

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
a few small changes!

**Comment by justinsong1608** [comment]
Screenshot 2023-01-24 at 10 36 55 AM

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+      <div>
+        <select name="reasons">
+          <option value="select-an-option"> Select an option</option>
+          <option value="because-it's-interesting">Because it's interesting</option>
+          <option value="because-it's-fun">Because it's fun</option>
+          <option value="because-it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <br>
```
Comment (JenBowman1) [inline html-forms/index.html]: Hi Justin ! please only use br elements when splitting up paragraphs of text onto new lines. Use divs to separate form elements onto new lines

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
one small change

**Comment by justinsong1608** [comment]
Let me fix the issue. I accidentally sent the wrong one!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**